src/event/writer.rs

```diff
@@ -93,6 +97,7 @@ impl EventWriter {
         EventWriter {
             writer_id,
             sender: tx,
+            event_handles: VecDeque::new(),
```

Comment:
I don't see the completed handles being removed. It would be good to do this is write_event after the write to prevent this from growing forever in memory if the user never calls flush.

Reply:
Now, it will remove the receivers when completed.